src/index.d.ts

```diff
@@ -251,15 +251,29 @@ declare namespace RamdaAdjunct {
         isNotNumber(val: any): boolean;
 
         /**
-         * Checks if value is a positive `Number` primitive or object.
+         * Checks if value is a positive `Number` primitive or object. Zero is not considered
+         * negative.
```

Comment:
Checks if value is a positive `Number` primitive or object. Zero is considered neither positive nor negative.